.openpublishing.redirection.json

```diff
@@ -13752,12 +13752,12 @@
     },
     {
       "source_path": "docs/c-runtime-library/operator-delete-crt.md",
-      "redirect_url": "docs/c-runtime-library/delete-operator-crt",
+      "redirect_url": "/cpp/c-runtime-library/delete-operator-crt",
       "redirect_document_id": false
     },
     {
       "source_path": "docs/c-runtime-library/operator-new-crt.md",
-      "redirect_url": "docs/c-runtime-library/new-operator-crt",
+      "redirect_url": "/cpp/c-runtime-library/new-operator-crt",
```

Comment:
The old `redirect_url` redirects `https://learn.microsoft.com/cpp/c-runtime-library/operator-new-crt` to `https://learn.microsoft.com/en-us/cpp/c-runtime-library/docs/c-runtime-library/new-operator-crt`, leading to a 404.